readme.md

```diff
@@ -208,11 +229,18 @@ $service->loadIdentifier('Authentication.Orm', [
 ]);
 
 // Load the authenticators
-$service->loadAuthenticator('Authentication.Session');
-$service->loadAuthenticator('Authentication.Form');
+$service->loadAuthenticator('Authentication.Session', [
+    'sessionKey' => 'Auth'
+]);
+$service->loadAuthenticator('Authentication.Form', [
+    'fields' => [
+        'username' => 'email',
+        'password' => 'password'
+    ]
+]);
 ```
 
-While this seems to be a little more to write, the benefit is a greater flexibility and better [separation of concerns](https://en.wikipedia.org/wiki/Separation_of_concerns).
+While this seems to be a little more to write, the benefit is a greater flexibility and better [separation of concerns](https://en.wikipedia.org/wiki/Separation_of_concerns). Plus, with this separation, it gives you much greater flexibility in developing your own authorization layer.
```

Comment:
@davidyell if you want to help coming up with a good authorization solution check the CakePHP issues, [there is already a ticket](https://github.com/cakephp/cakephp/issues/9835) for it.

You might want to take a look at [this](https://laravel.com/docs/5.4/authorization), [this](https://github.com/CanCanCommunity/cancancan) and [this](https://en.wikipedia.org/wiki/Authorization). Spending a good amount of planning the architecture and API is the key here. I have this on my agenda but before I'll have to finish the cookies and the cookie authenticator. **After** that I'm going to come back to authorization.